### ISSUE TITLE
Parse error on ternary operator with arrow function

### ISSUE THREAD

**Issue by hcomnetworkers** [SonarSource/SonarJS#859]
**I want to report a bug.**

**SonarJS version:** 3.3 (Build 5702)<br>
**SonarQube version:** 6.7 (Build 33306)<br>

**Rule key:** javascript:ParsingError

**Reproducer** 
```
const callback = action === 'shouldInc' ? inc : (v) => v; // False-positive here
```

**Expected behavior**
As I have tested in Browsers and checked in Babel, this seems to be a valid expression and no parse error.

**Comment by vilchik-elena** [comment]
```
a ? b : c => c
      ^ arrow function parameter
           ^  arrow function return type
```
